client/signup/steps/domain-upsell/index.jsx

```diff
@@ -0,0 +1,191 @@
+/**
+ * External dependencies
+ */
+
+import { useSelector } from 'react-redux';
+import React, { useEffect, useState } from 'react';
+import { useTranslate } from 'i18n-calypso';
+
+import StepWrapper from 'calypso/signup/step-wrapper';
+import {
+	getSecureYourBrand,
+	isRequestingSecureYourBrand,
+} from 'calypso/state/secure-your-brand/selectors';
+import { Button, Card } from '@automattic/components';
+import formatCurrency from '@automattic/format-currency';
+import QuerySecureYourBrand from 'calypso/components/data/query-secure-your-brand';
+import Badge from 'calypso/components/badge';
+import { domainRegistration } from 'calypso/lib/cart-values/cart-items';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+import FormFieldset from 'calypso/components/forms/form-fieldset';
+import FormRadio from 'calypso/components/forms/form-radio';
+import FormLabel from 'calypso/components/forms/form-label';
+
+export default function DomainUpsellStep( props ) {
+	const translate = useTranslate();
+	const {
+		flowName,
+		stepName,
+		positionInFlow,
+		signupDependencies: { siteSlug },
+	} = props;
+
+	const subHeaderText = translate(
+		'You can select an easier-to-remember name below at a one-time discounted price:'
+	);
+	const headerText = translate( 'Congrats, %s is almost ready!', {
+		args: [ siteSlug ],
+	} );
+
+	return (
+		<div className="domain-upsell__step-secton-wrapper">
+			<StepWrapper
+				flowName={ flowName }
+				stepName={ stepName }
+				positionInFlow={ positionInFlow }
+				headerText={ headerText }
+				fallbackHeaderText={ headerText }
+				subHeaderText={ subHeaderText }
+				fallbackSubHeaderText={ subHeaderText }
+				isWideLayout={ false }
+				stepContent={ RecommendedDomains( props ) }
+			/>
+		</div>
+	);
+}
+
+function getDomainName( siteSlug ) {
+	return siteSlug.replace( '.wordpress.com', '.blog' );
+}
+
+function FormattedSuggestion( translate, suggestion, isRecommended ) {
+	const currency = suggestion.currency;
+	return (
+		<div className="domain-upsell__domain-radio-label">
+			<div className="domain-upsell__domain-radio-label-domain-wrapper">
+				<div className="domain-upsell__domain-radio-label-domain">{ suggestion.domain }</div>
+				{ isRecommended && <Badge type="success">{ translate( 'Recommended' ) }</Badge> }
+			</div>
+			<div className="domain-upsell__registration-fee">
+				<div>
+					{ translate( 'Registration fee: {{strong}}%(originalCost)s{{/strong}}', {
+						args: {
+							originalCost: formatCurrency( suggestion.cost, currency, { stripZeros: true } ),
+						},
+						components: { strike: <strike />, strong: <strong /> },
+					} ) }
+				</div>
+				<small>
+					{ translate( 'Renews at: %(cost)s / year', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **22** times:
`translate( 'Renews at %(cost)s / year' )` *ES Score: 9*